app/views/group_members/new.html.erb

```diff
@@ -1,22 +1,41 @@
-<% set_page_title(t(".title")) %>
+<% if show_role_options %>
+  <% set_page_title(t(".title_show_role_options")) %>
+<% else %>
+  <% set_page_title(t(".title")) %>
+<% end %>
 <% content_for :back_link, govuk_back_link_to(group_members_path(@group)) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 id="heading" class="govuk-heading-l">
```

Comment:
The error summary should come above the page heading. On other pages, we put the `h1` inside the `form` to achieve this.

![Screenshot 2024-05-16 at 11 02 54](https://github.com/alphagov/forms-admin/assets/5648592/c697ef2c-19ac-4ab3-ac01-3504cd19cbd6)

Reply:
Thanks! sorry wasn't thinking properly, I've updated it.